### PR TITLE
757 print only finding upto 3

### PR DESCRIPTION
**Pull request by ShiriMoran**

### CODE REVIEW

File: pkg/linter/linterTypes.go [hunk 1]
```diff
@@ -67,15 +69,23 @@ func (lint *basicLinter) enableByDefault() bool {
...
+		findingRes = findingsResAll[:numFindingToPrint]
+		suffix = fmt.Sprintf("\n...and %d more\n", len(lint.findings)-numFindingToPrint)
+	} else {
+		findingRes = findingsResAll
 	}
-	sort.Strings(findingsRes)
 	header := fmt.Sprintf("%q %s\n", lintDesc, issues) +
-		strings.Repeat("~", len(lintDesc)+len(issues)+3) + "\n"
-	return header + strings.Join(findingsRes, "\n")
+		strings.Repeat("~", len(lintDesc)+len(issues)+numFindingToPrint) + "\n"
```
Comment (zivnevo) [inline pkg/linter/linterTypes.go:87]: Move the `issues` const to this file

Comment (zivnevo) [inline pkg/linter/linterTypes.go:87]:
```golang
	header := fmt.Sprintf("%q %s\n", lintDesc, issues)
	header += strings.Repeat("~", len(header)-1) + "\n"
```

**Review by zivnevo — CHANGES_REQUESTED** [review]

File: pkg/linter/linterTypes.go (hunk 1 above)
Comment (zivnevo) [reply pkg/linter/linterTypes.go:87]: or maybe the `issues` const is no longer needed

File: pkg/linter/linterTypes.go [hunk 2]
```diff
@@ -67,15 +69,23 @@ func (lint *basicLinter) enableByDefault() bool {
...
 	for i, thisFinding := range lint.findings {
-		findingsRes[i] = thisFinding.string()
+		findingsResAll[i] = thisFinding.string()
+	}
+	sort.Strings(findingsResAll)
+	var suffix string
+	var findingRes []string
+	if !printAll && len(lint.findings) > numFindingToPrint {
+		findingRes = findingsResAll[:numFindingToPrint]
+		suffix = fmt.Sprintf("\n...and %d more\n", len(lint.findings)-numFindingToPrint)
```
[the remark this answers is not in the document]
Comment (ShiriMoran) [reply pkg/linter/linterTypes.go]: done

File: pkg/linter/linterTypes.go (hunk 1 above)
Comment (ShiriMoran) [reply pkg/linter/linterTypes.go]: right. done

**Review by zivnevo — APPROVED** [review]

**Pull request merged**